README.md

```diff
@@ -53,6 +53,7 @@ _In alphabetical order and including links to external repository-based document
 - [Security](security/)
     - [GPG and Blackbox](security/gpg/blackbox.md)
     - [Crypto checklist](https://bit.ly/cryptochecklist)
+- [Accessibility](https://github.com/datamade/accessibility)
```

Comment:
this takes me to a page not found. looking at the other links in this section, does it just need to be the relative path? maybe [Accessibility](accessibility.README.md)

Reply:
good catch. fixed